solutions/devsprint-firas-safa-1/GitHubApp/Screens/RepositoryCell/RepositoryCell.swift

```diff
@@ -0,0 +1,64 @@
+import UIKit
+
+struct RepositoryCellViewConfiguration {
+    var title: String
+    var subtitle: String
+    var accessoryType: UITableViewCell.AccessoryType = .disclosureIndicator
+}
+
+class RepositoryCell: UITableViewCell {
+    
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        setupCellLayout()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    private lazy var mainTitle: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.font = .boldSystemFont(ofSize: 15)
+        return label
+    }()
```

Comment:
let mainTitle = UILabel()

- I would add it as a subview inside of the setup phase
- I would style it inside of the styling phase
- I would layout it in the layout case